src/actions/ui.js

```diff
@@ -119,8 +120,14 @@ export function clearHighlightLineRange() {
 }
 
 export function toggleConditionalBreakpointPanel(line: null | number) {
```

Comment:
@nyrosmith the flow issue had to do with the line input
This should fix it `export function toggleConditionalBreakpointPanel(line?: number) {`

Reply:
Thanks @bomsy for looking at it and helping me out!